maths/harshad_numbers.py

```diff
@@ -0,0 +1,132 @@
+"""
+A Harshad number is divisible by the sum of its digits in any base n.

```

Comment:
Nitpicks:
1) "Harshad" should be lowercase ("harshad" is not a name).
2) What you're describing here is an _all-harshad_ number. In general, a harshad number (or more specifically an n-harshad number) is a number that's divisible by the sum of its digits _in some given base n._